recipes/el-secretario

```diff
@@ -0,0 +1,7 @@
+(el-secretario
+ :fetcher git
+ :url "https://git.sr.ht/~zetagon/el-secretario"
+ :files ("el-secretario.el"
+         "el-secretario-source.el"
+         "el-secretario-message.el"
+         "el-secretario-function.el"))
```

Comment:
I believe this recipe has already been merged, so skipping this one.

Reply:
Sure, I'll just remove it from the PR.